src/functionalTest/java/uk/gov/hmcts/reform/wataskmonitor/MonitorTaskJobControllerForInitiationJobTest.java

```diff
@@ -62,9 +64,92 @@ public void task_initiation_job_should_initiate_task_and_taskState_should_be_una
             .statusCode(HttpStatus.OK.value())
             .body(is(expectedResponse.apply(JobName.INITIATION.name())));
 
-        Map<String, CamundaVariable> camundaVariableMap =
+        Map<String, CamundaVariable> camundaVariables =
             common.getTaskVariables(authenticationHeaders, taskVariables.getTaskId());
-        assertEquals("unassigned", camundaVariableMap.get("taskState").getValue());
+
+        assertEquals(taskVariables.getCaseId(), camundaVariables.get("caseId").getValue());
+        assertEquals("unassigned", camundaVariables.get("taskState").getValue());
+
+    }
+
+    @Test
+    public void task_initiation_job_should_not_initiate_delayed_task_and_taskState_should_be_unconfigured() {
+
+        TestVariables taskVariables = common.setupDelayedTaskAndRetrieveIds();
+
+        assertNotNull(taskVariables);
+        assertNotNull(taskVariables.getCaseId());
+        assertNotNull(taskVariables.getTaskId());
+
+        caseIds.add(taskVariables.getCaseId());
+
+        given()
+            .contentType(APPLICATION_JSON_VALUE)
+            .header(SERVICE_AUTHORIZATION, serviceToken)
+            .body(TestUtility.asJsonString(new MonitorTaskJobRequest(new JobDetails(JobName.INITIATION))))
+            .when()
+            .post("/monitor/tasks/jobs")
+            .then()
+            .statusCode(HttpStatus.OK.value())
+            .body(is(expectedResponse.apply(JobName.INITIATION.name())));
+
+        Map<String, CamundaVariable> camundaVariables =
+            common.getTaskVariablesFromCamunda(authenticationHeaders, taskVariables.getTaskId());
+
+        String caseId = ((HashMap) (((HashMap) camundaVariables).get("caseId"))).get("value").toString();
+        String actualTaskState = ((HashMap) (((HashMap) camundaVariables).get("taskState"))).get("value").toString();
```

Comment:
I don't think these castings are needed you should be able to do something like:
`camundaVariables.get("caseId").getValue()`

since you have a `Map<String, CamundaVariable>`